### PR TITLE
fix(keywords): updates categories and keywords

### PR DESCRIPTION
**Pull request by invincibleJai**
updates categories and keywords.

- Details of values which can go in categories can be seen here https://code.visualstudio.com/docs/extensionAPI/extension-manifest

- keywords should be not more than 5

### CODE REVIEW

**Comment by sivaavkd** [comment]
In computer programming, a linter is a program that scans source code and flags any constructs that are likely to be bugs - So, I suggest that our extension can be put under the category, Linters.
Tags look good to me.

**Comment by invincibleJai** [comment]
thanks @sivaavkd any inputs for auth one https://github.com/fabric8-analytics/vscode-osio-auth/blob/master/package.json#L34-L38

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   ],
   "keywords": [
     "OpenShift.io",
-    "fabric8-analytics",
+    "CVE",
```
Comment (samuzzal-choudhury) [inline package.json]: @invincibleJai Following categories are the ones that I think are appropriate for this extension:
```
Programming Languages
Linters
Other
```

IMO, here are the five keywords that we can associate with the extension:
```
"analytics"
"dependency recommender"
"cve security vulnerability"
"license "
"node maven"
```

Also, the title description requires a change. I think "Flags security vulnerabilities, reports license anomalies and recommends dependencies for NodeJS and Maven applications." is a good way to present what this extension does. Please opine @krishnapaparaju @miteshvp @sivaavkd

**Comment by samuzzal-choudhury** [comment]
@invincibleJai please merge this PR after the build is through.

**Comment by invincibleJai** [comment]
thx @samuzzal-choudhury :)

**Pull request merged**